transport-msg.tex

```diff
@@ -1024,22 +1093,27 @@ \subsection{Bus Messages}\label{sec:Virtio Transport Options / Virtio Over Messa
 \hline
 Reserved                    & 0x0  &        \\
 \hline
-\busref{ERROR}              & 0x1  & Any    \\
+Reserved                    & 0x1  &        \\
 \hline
 \busref{GET_DEVICES}        & 0x2  & Driver \\
 \hline
-\busref{DEVICE_ADDED}       & 0x3  & Device \\
-\hline
-\busref{DEVICE_REMOVED}     & 0x4  & Device \\
+\busref{PING}               & 0x3  & Any    \\
 \hline
-\busref{PING}               & 0x5  & Any    \\
+\busref{DEVICE_ADDED}       & 0x41 & Device \\
 \hline
-\busref{STATUS}             & 0x6  & Driver  \\
+\busref{DEVICE_REMOVED}     & 0x42 & Device \\
 \hline
 \end{tabular}
 
-Bus message ID 6-127 are reserved for future use in the specification and
-message ID 128-255 MAY be used for implementation specific needs.
+Bus message IDs below 0x80 are reserved for standardizes (but optional) bus
+messages.  A few are used here and more are expected in the future. Bus message
+IDs below 0x40 are used for messages that require a response and 0x40 and above
+have no response.
+
+Bus message IDs 0x80 and above are bus implementation specific. Bus
+implementations \emph{MAY} specify the policy that IDs below 0xC0 be used
+for messages with a response and IDs 0xC0 and above are used fro messages with
+no response.
 
 \paragraph{Note:} A bus implementation \textbf{is not required} to use these
```

Comment:
This is appearing as a numbered chapter in the pdf. We might need to use something else than paragraph here